src/libraries/Microsoft.NETCore.Platforms/src/Microsoft.NETCore.Platforms.csproj

```diff
@@ -1,9 +1,40 @@
-<Project Sdk="Microsoft.Build.NoTargets">
+<Project Sdk="Microsoft.NET.SDK">
   <!-- These are wrapper project files for packaging.-->
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0</TargetFrameworks>
-    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
-    <!-- Avoid trying to binplace non existent documentation file.-->
-    <EnableBinPlacing>false</EnableBinPlacing>
+    <TargetFrameworks>$(NetCoreAppToolCurrent);net472</TargetFrameworks>
+    <PackageId>$(MSBuildProjectName)</PackageId>
```

Comment:
Isn't this the default in the SDK?

Reply:
oh I see you are changing AssemblyName bellow to be called BuildTasks, disregard.